components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m

```diff
@@ -186,4 +186,17 @@ - (void)testWithLongMessageLongActionRTLArabic {
   [self generateSnapshotAndVerifyForView:messageView];
 }
 
+- (void)testWithCustomElevation {
+  // Given
+  MDCSnackbarMessageView *messageView = [self snackbarMessageViewWithText:kItemTitleShort1Latin
+                                                              actionTitle:kItemTitleShort2Latin];
+  messageView.frame = CGRectMake(0, 0, kWidth, kHeightSingleLineText);
+
+  // When
+  messageView.elevation = 0;
```

Comment:
Please either add a second test for custom elevation with a non-zero value or change this to something other than zero.  If there were a bug in setting the elevation that caused the shadow elevation to become invalid (and perhaps reset to `0`), it would not be possible to detect in this test.

Reply:
Done